yt/data_objects/tests/test_fluxes.py

```diff
@@ -115,10 +109,18 @@ def test_correct_output_unit():
     sur = ds.surface(sp1,"HI_Density", .5*Nmax)
     sur['x'][0]
 
-@requires_file(ISOGAL)
+def test_correct_output_unit_fake_ds():
+    # implementing test_correct_output_unit() with fake dataset
```

Comment:
can you delete this comment but retain the comment in the original test referencing #1368?

Reply:
done